x-pack/filebeat/input/awss3/input_integration_test.go

```diff
@@ -328,11 +325,19 @@ func uploadS3TestFiles(t *testing.T, region, bucket string, filenames ...string)
 			t.Fatalf("Failed to open file %q, %v", filename, err)
 		}
 
+		contentType := ""
+		if strings.HasSuffix(filename, "ndjson") || strings.HasSuffix(filename, "ndjson.gz") {
+			contentType = "let-CI-fail-" + contentTypeNDJSON + "; charset=UTF-8"
```

Comment:
what is the reason for this `let-CI-fail`?

Reply:
I wanted to asses that the integration tests were run in CI, forcing a failure

they don't: CI is green anyway 

(having `let-CI-fail-` has suffix https://github.com/elastic/beats/pull/32767/files#diff-f345fd6a1f5ea9523117d4ead2e5f1d13fb82eb1c65a089fd34fcdd514916a96R156 will be false)